sgl-kernel/python/sgl_kernel/gemm.py

```diff
@@ -73,7 +73,7 @@ def bmm_fp8(
     out: Optional[torch.Tensor] = None,
 ) -> torch.Tensor:
     if out is None:
-        out = torch.empty(
+        out = torch.zeros(
```

Comment:
I am not appreciate that introducing extra zeros kernel in model forward phase. Can we double check whether these zeros is necessary？